src/components/molecules/definition/definition.scss

```diff
@@ -0,0 +1,59 @@
+#dictionary-definition {
```

Comment:
Try not to use ids in css(makes it harder to overcome), it's already a .definition so why not use that as the landmark?

Reply:
I actually absolutely refrain from using ids to style, but I was also looking to match current prod as much as possible to avoid issues. Although we no longer have `best-bet-definition` id felt a name swap would be easier that's why I went with this. Would gladly remove this.

Reply:
Done!